### PR TITLE
Adding vSphere IPI config files

### PR DESCRIPTION
**Pull request by vavuthu**
Signed-off-by: vavuthu <vavuthu@redhat.com>

### CODE REVIEW

File: conf/deployment/vsphere/ipi_1az_rhcos_vmfs_3m_3w.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+# This config is suppose to work on most of DCs we have.
```
Comment (prsurve) [inline conf/deployment/vsphere/ipi_1az_rhcos_vmfs_3m_3w.yaml:2]: Note Must be changed to `This config is suppose to work on DC-CP, DC-ECO, DC-PS`

Comment (vavuthu) [reply conf/deployment/vsphere/ipi_1az_rhcos_vmfs_3m_3w.yaml]: Done

**Review by prsurve — APPROVED** [review]

File: conf/deployment/vsphere/ipi_1az_rhcos_vmfs_3m_3w.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (ebenahar) [inline conf/deployment/vsphere/ipi_1az_rhcos_vmfs_3m_3w.yaml:1]: @vavuthu shouldn't it be vsan?

Comment (vavuthu) [reply conf/deployment/vsphere/ipi_1az_rhcos_vmfs_3m_3w.yaml:1]: nope, its for vmfs. we already have for vsan

**Review by clacroix12 — APPROVED** [review]

**Review by petr-balogh — APPROVED** [review]

**Pull request merged**